evm/broadcast-transactions.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const chalk = require('chalk');
+const { Command, Option } = require('commander');
+const { ethers } = require('hardhat');
+const {
+    providers: { JsonRpcProvider },
+} = ethers;
+const readlineSync = require('readline-sync');
+
+const { printError, printInfo, printObj } = require('./utils');
+const { sendTx, getAllSignersData, updateSignersData } = require('./offline-sign-utils');
+
+async function processTransactions(filePath, provider) {
+    try {
+        const signersData = await getAllSignersData(filePath);
```

Comment:
i'd like to keep this super simple for now. read file -> read chain config -> send tx -> log success/error